docs/sources/contribute-documentation/contribute-release-notes/index.md

```diff
@@ -1,73 +1,131 @@
 keywords:
   - what's new
   - release notes
+menuTitle: Contribute to What's new or release notes
+title: Contribute to What's new or release notes
+weight: 250
 ---
 
+<!-- vale Grafana.Timeless = NO -->
+
 # Contribute to What's new or release notes
 
 This topic explains the decisions and actions associated with collecting, writing, and publishing What's new content.
 
-{{% admonition type="note" %}}
+{{< admonition type="note" >}}
 This topic is only relevant for internal Grafana Labs contributors.
-{{% /admonition %}}
+{{< /admonition >}}
 
-## What's new doc development process
+## What's new documentation development process
 
-What’s new content is published to the website through the website content management system (CMS), and can be accessed [here](https://admin.grafana.com/content-admin/#/collections/whats-new).
+You publish What’s new content to the website through the website Content Management System (CMS).
+To add a new note, browse to the [What's new CMS collection](https://admin.grafana.com/content-admin/#/collections/whats-new).
 
-Because this platform is meant to be used by the entire organization, by default anyone can contribute and publish to What’s new, without the need for approval. **Quality assurance is a conversation within and between contributing teams and internal stakeholders**, but there are some best practice guidelines described in the last two sections of this topic.
+Because this platform is meant to be used by the entire organization, by default anyone can contribute and publish to What’s new, without the need for approval.
+Quality assurance is a conversation within and between contributing teams and internal stakeholders, but there are some best practice guidelines described in the last two sections of this topic.
 
-Release notes should be entered into the CMS 2-4 weeks before the feature is available, depending on the size of the product or feature. This gives the GTM team time for promotion and enablement. For Grafana versioned releases, have your content entered in the CMS by the cut-off date communicated by the delivery team. For more information, see the [RADS guidelines](https://grafana-intranet--simpplr.vf.force.com/apex/simpplr__app?u=/site/a145f000001dCXBAA2/page/a125f000004oOF7AAM).
+You should enter release notes into the CMS two to four weeks before the feature is available, depending on the size of the product or feature.
+This gives the Go To Market (GTM) team time for promotion and enablement.
+For Grafana versioned releases, have your content entered in the CMS by the cut-off date communicated by the delivery team.
+For more information, refer to the [Record Announce Document Ship (RADS) guidelines](https://grafana-intranet--simpplr.vf.force.com/apex/simpplr__app?u=/site/a145f000001dCXBAA2/page/a125f000004oOF7AAM).
 
-Note that in the context of the CMS, the word "published" has a slightly different meaning than in general use:
+{{< admonition type="note" >}}
```

Comment:
I know I used the word "note" here, but I didn't intend for this to be in a note, because I don't want folks to skip this as something that's an aside that's not integral to the content. It's integral that folks understand this concept, so it should be part of the body text. We can use a different lead in like maybe "It's important to understand that..."

Reply:
Ah yeah, I was mostly copying the feedback I got from Clayton in https://github.com/grafana/writers-toolkit/pull/461#discussion_r1480346185.

I'll make the change to use a different lead.

Reply:
Done in [`1f68d0d` (#490)](https://github.com/grafana/writers-toolkit/pull/490/commits/1f68d0db9aa1156af345901c2db8972e0357dc39).